src/Sportalytics.Feed.Application/Handlers/GetSportEventByIdHandler.cs

```diff
@@ -13,7 +15,18 @@ public async Task<ResponseSportEventDto> Handle(GetSportEventByIdQuery request,
     {
         var id = request.SportId;
         var sportEventRepository = repositoryManager.SportEvents;
-        var sportEvent = await sportEventRepository.GetByIdAsync(id);
+
+
+        var sportEventFilter = new SportEventFilter
+        {
+            Ids = new[]
+            {
+                id
+            }
+        };
+
+        var sportEventQuery = sportEventRepository.Query(sportEventFilter);
+        var sportEvent = await sportEventQuery.FirstOrDefaultAsync(cancellationToken);
         if (sportEvent is null)
```

Comment:
To avoid adding if check for null, try to create extension method .EnsureFound that will do it and throw exception in case of null